dotCMS/src/main/java/com/dotcms/tika/TikaUtils.java

```diff
@@ -7,11 +7,14 @@
 import com.dotcms.repackage.org.apache.tika.io.TikaInputStream;
 import com.dotcms.repackage.org.apache.tika.metadata.Metadata;
 import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.exception.DotSecurityException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)